tests/server_send.cpp

```diff
@@ -25,6 +25,8 @@
 using namespace ioremap::elliptics;
 using namespace boost::unit_test;
 
+#define ENABLE_LONG_TESTS 0
```

Comment:
Ok, I will make it enabled by default and add cmake config option which will allow to disable such "long" tests.